action.yml

```diff
@@ -0,0 +1,18 @@
+name: "Download cursed OIDC token"
+description: "Download an unsafe OIDC token to ./oidc-token.txt. This is useful for sigstore conformance testing"
+
+permissions: {}
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Install dependencies
+      # this should be a no-op as it's already installed on GitHub runners
+      run: pip install pyjwt
+      shell: bash
```

Comment:
This is extremely minor, but I'd also be fine with dropping the `pyjwt` dep and just decoding the JWT manually (since this is all just for introspection + a cheap validity check).

Nonblocker, though.

Reply:
oh right. Did not think of that. So it's
* b64 decode of one of the sections (with manual padding I suppose)
* parse as json, extract "exp" field 

I'll test if its that simple...

Reply:
:+1: